packages/backend/src/plugins/catalog.ts

```diff
@@ -106,6 +108,18 @@ export default async function createPlugin(
     );
   }
 
+  if (isAapEnabled) {
+    builder.addEntityProvider(
+      AapResourceEntityProvider.fromConfig(env.config, {
+        logger: env.logger,
+        schedule: env.scheduler.createScheduledTaskRunner({
+          frequency: { minutes: 1 },
```

Comment:
All the other Entity Providers installed in the showcase currently use a frequency of 30 minutes or 1 hour, I think 1 minute might be a bit too frequent?